### PR TITLE
Lane carousels

### PR DESCRIPTION
**Pull request by lanebret85**
____ Wrote Tests ____ Implemented ____ Reviewed

Necessary checkmarks:

- [ ] All Tests are Passing
- [x] The code will run locally

Type of change

- [x] New feature
- [x] Bug Fix

Implements/Fixes:

    description closes #

Check the correct boxes

- [?] This broke nothing
- [?] This broke some stuff
- [?] This broke everything

Testing Changes

- [x] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened)

Checklist:

- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have fully tested my code

TY

### CODE REVIEW

File: .gitignore
Comment (reedhillmar) [inline .gitignore:1]: Everything doubled here?

File: app/controllers/api/v1/board_games_controller.rb
Comment (reedhillmar) [inline app/controllers/api/v1/board_games_controller.rb:1]: NOOO! Why did all_by_params and filtering get deleted!?!

File: config/routes.rb
Comment (reedhillmar) [inline config/routes.rb:1]: Why does it say no merge conflicts when there are _clearly_ merge confilcts?

**Review by reedhillmar — CHANGES_REQUESTED** [review]
It says no merge conflicts, but there are merge conflicts to resolve...

File: .gitignore
Comment (lanebret85) [reply .gitignore:1]: No longer doubled! Good eye!

File: app/controllers/api/v1/board_games_controller.rb
Comment (lanebret85) [reply app/controllers/api/v1/board_games_controller.rb:1]: Oops... stupid merging. Should be fixed now!

File: config/routes.rb
Comment (lanebret85) [reply config/routes.rb:1]: I feel 🤬 about this

Comment (lanebret85) [reply config/routes.rb:1]: The merge conflicts it _should_ have alerted us about are fixed now.

**Review by ConnorRichmond — APPROVED** [review]
I didnt see any issues. I dont know what youre talking about

**Review by reedhillmar — APPROVED** [review]
Thanks for resolving those merge conflicts, approved to merge!

**Pull request merged**